plugs/message/html/render/about.js

```diff
@@ -17,67 +17,80 @@ exports.needs = nest({
+          }
+        }))
+      }
 
-    if (c.name) {
-      var target = api.profile.html.person(c.about, c.name)
-      miniContent.push(computed([self, api.about.obs.name(c.about), c.name], (self, a, b) => {
-        if (self) {
-          return [i18n('self identifies as "'), target, '"']
-        } else if (a === b) {
-          return [i18n('identified '), api.profile.html.person(c.about)]
-        } else {
-          return [i18n('identifies '), api.profile.html.person(c.about), i18n(' as "'), target, '"']
+      if (c.image) {
+        if (!miniContent.length) {
+          var imageAction = self ? i18n('self assigned a display image') : [i18n('assigned a display image to '), api.profile.html.person(c.about)]
+          miniContent.push(imageAction)
         }
-      }))
-    }
 
-    if (c.image) {
-      if (!miniContent.length) {
-        var imageAction = self ? i18n('self assigned a display image') : [i18n('assigned a display image to '), api.profile.html.person(c.about)]
-        miniContent.push(imageAction)
+        content.push(h('a AboutImage', {
+          href: c.about
+        }, [
+          h('img', {src: api.blob.sync.url(c.image)})
+        ]))
       }
 
-      content.push(h('a AboutImage', {
-        href: c.about
-      }, [
-        h('img', {src: api.blob.sync.url(c.image)})
-      ]))
-    }
+      var elements = []
 
-    var elements = []
+      if (miniContent.length) {
+        var element = api.message.html.layout(msg, extend({
+          showActions: true,
+          miniContent,
+          content,
+          layout: 'mini'
+        }, opts))
+        elements.push(api.message.html.decorate(element, { msg }))
+      }
 
-    if (miniContent.length) {
-      var element = api.message.html.layout(msg, extend({
-        showActions: true,
-        miniContent,
-        content,
-        layout: 'mini'
-      }, opts))
-      elements.push(api.message.html.decorate(element, { msg }))
-    }
+      if (c.description) {
+        elements.push(api.message.html.decorate(api.message.html.layout(msg, extend({
+          showActions: true,
+          miniContent: self ? i18n('self assigned a description') : [i18n('assigned a description to '), api.profile.html.person(c.about)],
+          content: api.message.html.markdown(c.description),
+          layout: 'mini'
+        }, opts)), { msg }))
+      }
 
-    if (c.description) {
-      elements.push(api.message.html.decorate(api.message.html.layout(msg, extend({
-        showActions: true,
-        miniContent: self ? i18n('self assigned a description') : [i18n('assigned a description to '), api.profile.html.person(c.about)],
-        content: api.message.html.markdown(c.description),
-        layout: 'mini'
-      }, opts)), { msg }))
+      return elements
     }
-
-    return elements
   })
+
+  function isRenderable(msg) {
+    if (msg.value.content.type !== 'about') return false
```

Comment:
@Happy0 Something you might want to consider is that if you return `false` from a depject `"first"`, that will be final answer. You need to return `undefined` or nothing at all if you want it to continue to the next one.

Reply:
Thanks! :)